src/popup/router/components/FungibleTokens/TokensListItem.vue

```diff
@@ -20,30 +22,42 @@
       />
     </div>
     <div
-      v-if="tokenData.contractId == 'aeternity'"
+      v-if="tokenData.contractId === 'aeternity'"
       class="row"
     >
       <div class="price">
-        @ {{ formatCurrency(aePublicData.current_price) }}
+        @ {{ getPrice }}
       </div>
       <div class="price">
         {{ convertToCurrencyFormatted(tokenData.convertedBalance) }}
       </div>
     </div>
-  </RouterLink>
+  </Component>
 </template>
 
 <script>
 import { mapGetters, mapState } from 'vuex';
+import BigNumber from 'bignumber.js';
 import TokenAmount from '../TokenAmount.vue';
 import Tokens from '../Tokens.vue';
 
 export default {
   components: { TokenAmount, Tokens },
-  props: { tokenData: { type: Object, default: null } },
+  props: {
+    tokenData: { type: Object, default: null },
+    preventNavigation: { type: Boolean },
```

Comment:
You can use shorthand:
```suggestion
    preventNavigation: Boolean,
```